CameraXBasic/app/src/main/java/com/android/example/cameraxbasic/utils/ViewExtensions.kt

```diff
@@ -75,16 +75,16 @@ fun View.padWithDisplayCutout() {
 /** Same as [AlertDialog.show] but setting immersive mode in the dialog's window */
 fun AlertDialog.showImmersive() {
     // Set the dialog to not focusable
-    window.setFlags(
+    window?.setFlags(
```

Comment:
@SubhrajyotiSen I think that you could simplify code using `nullable.let {   }` to to remove doubled `window?...`

Reply:
Good point